### PR TITLE
Trend Request

### PR DESCRIPTION
**Pull request by tobbhie**

### CODE REVIEW

**Comment by usmansbk** [comment]
Pull latest changes

File: src/routes/requestRoute.js [hunk 1]
```diff
@@ -3,11 +3,20 @@ const authenticate = require("../middlewares/authenticatorMiddleware");
 const RequestCtrl = require("./../controllers/RequestController");
 const AdminRequestCtrl = require("./../controllers/admin/RequestController");
 
+
 module.exports = () => {
+<<<<<<< HEAD
```
Comment (usmansbk) [inline src/routes/requestRoute.js:8]: You didn't resolve your conflicts

File: package.json [hunk 2]
```diff
@@ -34,8 +34,5 @@
     "mongodb-memory-server": "^6.6.1",
     "nodemon": "^1.19.4",
     "supertest": "^4.0.2"
```
Comment (usmansbk) [inline package.json:36]: This has been fixed in the latest codebase

File: src/routes/requestRoute.js (hunk 1 above)
Comment (tobbhie) [reply src/routes/requestRoute.js:8]: okay, I did a couple of merge conflicts even before making a PR though

**Pull request closed without merging**